docs/scenarios_authoring/Enrichers.md

```diff
@@ -41,6 +41,28 @@ DatabaseLookupEnricher is a specialized look-up component; it returns all column
 
 ![alt_text](img/databaseLookupEnricher.png "databaseLookup Enricher")
 
+### DecisionTable enricher
+
+This component allows the user to match input to a pre-defined result. Please be wary that this component works a bit different than a classical decision table([Decision tables in Wikipedia](https://en.wikipedia.org/wiki/Decision_table)). It consists of the fields:
+- Name: the name of the component.
+- Decision Table: a user defined table of type TabularTypedData, discussed below.
+- Match condition: a Boolean SpEL expression that matches input to table rows, discussed below.
+- Output: name of the output variable which will hold data generated by the component.
+- Description: user-defined description of the particular component. 
+
+The component has two main elements: _Decision Table_ which is a user-defined table of values and _Match condition_ - a SpEL expression which Nu will use to match the input to the table values.
```

Comment:
I'd split the component doc into two parts: the description and the example.
In the example part it'd be great if we could say something about:

1. node input (showing the example of the processing input)
2. the decision table node configuration:
  a) what we have in the table
  b) what we have in the match condition
3. node output (showing the example of the output)

Reply:
I have added an input screenshot. I'm trying to add an output screenshot but staging keeps dying on ad hoc testing ;_;